test/cluster/test_ttl_row.py

```
@@ -346,6 +346,101 @@ async def test_row_ttl_multi_dc(manager: ManagerClient):
+                SimpleStatement(f'SELECT p FROM {table}', consistency_level=ConsistencyLevel.LOCAL_QUORUM))))
+            assert remaining_with_dc2_down < 50, "Expected some items to expire with DC2 down"
+
+            # Bring DC2 back up. Now all nodes can participate in TTL scanning
+            # again, so the remaining items (those whose global primary was in
+            # DC2) should also expire.
+            for server in dc2_servers:
+                await manager.server_start(server.server_id)
+
+            timeout = time.time() + 120
+            while time.time() < timeout:
+                if 0 == len(list(await cql.run_async(SimpleStatement(f'SELECT p FROM {table}', consistency_level=ConsistencyLevel.QUORUM)))):
```

Ruff (0.16.1)

[warning] Yoda condition detected

(SIM300)